elasticsearch/src/main/java/io/micronaut/elasticsearch/DefaultElasticsearchClientFactory.java

```diff
@@ -54,6 +63,18 @@ RestClient restClient(DefaultElasticsearchConfigurationProperties elasticsearchC
         return restClientBuilder(elasticsearchConfiguration).build();
     }
 
+    /**
+     * @param elasticsearchConfiguration The {@link DefaultElasticsearchConfigurationProperties} object
+     * @return The Elasticsearch Client
```

Comment:
Needs `@since` and please bump the minor version of the project

Reply:
bumped to 4.1.1